src/sidebar/components/menu-section.js

```diff
@@ -38,8 +38,5 @@ MenuSection.propTypes = {
   /**
    * Menu items to display in this section.
    */
-  children: propTypes.oneOfType([
-    propTypes.object,
-    propTypes.arrayOf(propTypes.object),
-  ]).isRequired,
+  children: propTypes.any.isRequired,
```

Comment:
Can you help me understand this change? Was the previous `propTypes` value for this property problematic with the test changes or problematic in general?

Reply:
If one of the children is conditionally rendered, and that condition is false, then the `children` array will have a `boolean` in it.